### DISCUSSION TITLE
重启服务器后FTP服务启动失败、手动启动失败

### DISCUSSION THREAD

**Discussion by ahong18s** [1Panel-dev/1Panel#6861] [Q&A]
### 1Panel 版本

1.10.20 LTS

### 问题描述

1. FTP启动失败。
2. 另外，安装"病毒扫描"后clamav无法启动，也没有报错，直到重启服务器后自动启动。

### 重现步骤

1. 安装FTP
2. 重启服务器，或停止后启动
3. 查看FTP状态、手动启动FTP

### 期待的正确结果

重启服务器后，正常启动FTP服务

### 相关日志输出

```shell
服务内部错误: start the pure-ftpd.service failed, err: stderr: Job for pure-ftpd.service failed because the control process exited with error code. See "systemctl status pure-ftpd.service" and "journalctl -xeu pure-ftpd.service" for details.
```

### 附加信息

截图：
https://github.com/ahong18s/ISSUE/blob/master/1panel/%E9%87%8D%E5%90%AF%E5%90%8EFTP%E5%90%AF%E5%8A%A8%E5%A4%B1%E8%B4%A5_2024-10-26_21-40-54.png

**Comment by wanghe-fit2cloud** [comment]
> See "systemctl status pure-ftpd.service" and "journalctl -xeu pure-ftpd.service" for details.

可以根据提示命令先查看详细日志。  
> 这可能与服务器操作系统有关，建议参考相应系统的 FTP 安装教程。

**Reply by ahong18s** [reply]
> > 有关详细信息，请参阅 “systemctl status pure-ftpd.service” 和 “journalctl -xeu pure-ftpd.service”。
> 
> 可以根据提示命令先查看详细日志。
> 
> > 这可能与服务器操作系统有关，建议参考相应系统的 FTP 安装教程。

本月初在云服务器、本地服务器的相同环境下安装 FTP，无任何问题。

已经回滚了，我在本地再复现一次吧。

**Reply by ahong18s** [reply]
> > > 有关详细信息，请参阅 “systemctl status pure-ftpd.service” 和 “journalctl -xeu pure-ftpd.service”。
> > 
> > 
> > 可以根据提示命令先查看详细日志。
> > > 这可能与服务器操作系统有关，建议参考相应系统的 FTP 安装教程。
> 
> 本月初在云服务器、本地服务器的相同环境下安装 FTP，无任何问题。
> 
> 已经回滚了，我在本地再复现一次吧。

日志：

https://github.com/ahong18s/ISSUE/blob/master/1panel/%E5%AE%89%E8%A3%85FTP%E5%B9%B6%E9%87%8D%E5%90%AF%E6%9C%8D%E5%8A%A1%E5%99%A8%E5%90%8E-%E6%89%8B%E5%8A%A8%E5%90%AF%E5%8A%A8%E6%8A%A5%E9%94%99.txt

**Comment by ahong18s** [comment]
我注意到状态是未回复，晚上9点你们回复了就立刻改了状态，怎么回事？欠你们的？

**Reply by wanghe-fit2cloud** [reply]
![image](https://github.com/user-attachments/assets/64d4f5a3-66f5-48a9-9fbb-9998393bc7c2)
